main.go

```diff
@@ -286,26 +285,26 @@ func triggerBuild(conf *config, build *buildOptions) error {
 				return err
 			} else {
 				log.Infof("%s version %s is being used in %s: ", versionedRepo, version, build.baseBranch)
-				buildParameter.Add(repoToJenkinsParameter(versionedRepo), version)
+				buildParameters = append(buildParameters, &gitlab.PipelineVariable{repoToJenkinsParameter(versionedRepo), version})
```

Comment:
Maybe rename `repoToJenkinsParameter` to `repoToParameter`?

Reply:
I renamed it to `repoToBuildParameter`